File: esque-examples/esque-example-core-simple/src/main/java/org/loesak/esque/examples/simple/Application.java
package org.loesak.esque.examples.simple;

import org.apache.http.HttpHost;
import org.elasticsearch.client.RestClient;
import org.loesak.esque.core.Esque;

public class Application {

    public static void main(String... args) throws Exception {
        final String migrationKey = "esque-example-core-simple";

        try(Esque esque = new Esque(
                RestClient.builder(new HttpHost("localhost", 9200, "http"))
                          .build(),
                migrationKey)) {
            esque.execute();
        }
    }
}

File: esque-examples/esque-example-core-simple/src/main/kotlin/org/loesak/esque/examples/simple/Application.kt
package org.loesak.esque.examples.simple

import org.apache.http.HttpHost
import org.elasticsearch.client.RestClient
import org.loesak.esque.core.Esque
import org.loesak.esque.core.EsqueConfiguration

fun main() {
  Esque(
          RestClient.builder(HttpHost("localhost", 9200, "http")).build(),
          EsqueConfiguration(migrationKey = "esque-example-core-simple"),
      )
      .use { it.execute() }
}
